habitat/tasks/nav/nav_task.py

```diff
@@ -393,10 +393,11 @@ def reset_metric(self, episode):
 
     def update_metric(self, episode, action):
         if self._metric is None:
-            self._metric = 0
-
+            self._metric = {"count": 0, "is_collision": False}
```

Comment:
Using strings directly for keys is not idea, can you store the keys as some kind of class attribute, something like `self._collision_key = "is_collision"`